Makefile

```diff
@@ -11,20 +11,19 @@ CONFIGURATOR_SRC := $(shell find ./configurator/src) configurator/Cargo.toml con
 all: c-lightning.s9pk
 
 install: c-lightning.s9pk
-	appmgr install c-lightning.s9pk
+	embassy-sdk pack
```

Comment:
the install command can be deleted. instead, update the `bitcoind.s9pk` step below, like [here](https://github.com/Start9Labs/cups-wrapper/pull/1/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R19)

there needs to be both a `embassy-sdk pack` and `embassy-sdk verify` step